integration/shared/global/service_command_test.go

```diff
@@ -294,7 +294,7 @@ var _ = Describe("service command", func() {
 							Eventually(session).Should(Say("tags:\\s+database, email"))
 							Eventually(session).Should(Say("plan:\\s+%s", servicePlan))
 							Eventually(session).Should(Say("description:\\s+fake service"))
-							Eventually(session).Should(Say("documentation:"))
+							Eventually(session).Should(Say("documentation:\\s+http://documentation\\.url"))
```

Comment:
Could you please add this to line 346 below as well (and any other place that does an assertion on the documentation URL in this file)? Sorry for not mentioning this in the first review -- only just noticed it on the second glance.

Reply:
@henryaj @williammartin

Reply:
Thank you @jenspinney for pointing this our, it totally makes sense. I've added it.